packages/doc/content/components/components/card/eventcard.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Event Card'
```

Comment:
Why not just 'Event'?

Reply:
I've changed the Card menu to:

![image](https://user-images.githubusercontent.com/11219999/72634701-d48ab280-3939-11ea-807c-841ca7c3e135.png)